WhaleSpotting/react-app/src/tests/BannerImage.test.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { BannerImage } from "../components/BannerImage";
+import { BrowserRouter as Router } from "react-router-dom";
+
+test("Renders banner image", () => {
+    render(<Router><BannerImage/></Router>);
```

Comment:
can you remove the Router pls, and then tidy the imports

Reply:
addressed.